sct.py

```diff
@@ -677,6 +677,29 @@ def run_test(argv, backend, config, logdir):
                   failfast=False, buffer=False, catchbreak=True)
 
 
+@cli.command('run-pytest', help="Run tests using pytest")
+@click.argument('target')
+@click.option('-b', '--backend', type=click.Choice(SCTConfiguration.available_backends), help="Backend to use")
+@click.option('-c', '--config', multiple=True, type=click.Path(exists=True), help="Test config .yaml to use, can have multiple of those")
+@click.option('-l', '--logdir', help="Directory to use for logs")
+def run_pytest(target, backend, config, logdir):
+    if config:
+        os.environ['SCT_CONFIG_FILES'] = str(list(config))
+    if backend:
+        os.environ['SCT_CLUSTER_BACKEND'] = backend
+
+    if logdir:
+        os.environ['_SCT_LOGDIR'] = logdir
+
+    logfile = os.path.join(Setup.logdir(), 'output.log')
```

Comment:
Maybe include here test run specific part? So, we could distinguish different test runs logs in one single dir.

Reply:
It handles folders in the same manner as run-test, meaning that for each test run folder in ~/sct-results/ is going to be created.